core/cc_batch.go

```diff
@@ -134,7 +134,8 @@ func (cc *Chaincode) loadFromBatch(
 	}
 
 	sender := types.NewSenderFromAddr((*types.Address)(pending.GetSender()))
-	if err = checkNonce(stub, sender, pending.GetNonce()); err != nil {
+	n := &Nonce{}
```

Comment:
If you just allocating memory without initialization it would be "ideomatic" to use n := new(Nonce).

Reply:
fix